packages/vue/examples/search-box/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vue.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Reactivesearch | Searchbox inline ai response</title>
```

Comment:
@SavvyShah Can you correct the title here? It's a simple searchbox component, if you want to do a q-n-a demo then please update the App.vue file to pass enableAI prop to `true`